shared/actions/config/index.tsx

```diff
@@ -562,17 +562,11 @@ const updateServerConfig = async (state: Container.TypedState, action: ConfigGen
     const str = await RPCTypes.apiserverGetWithSessionRpcPromise({
       endpoint: 'user/features',
     })
-    const obj: {
-      features: {
-        admin?: {
-          value: boolean
-        }
-      }
-    } = JSON.parse(str.body)
-    const features = Object.keys(obj.features).reduce((map, key) => {
-      map[key] = obj.features[key as any]?.value ?? false
+    const obj: {features: any} = JSON.parse(str.body)
+    const features = Object.keys(obj.features).reduce<{[key: string]: boolean}>((map, key) => {
```

Comment:
Can you add a comment explaining what this is doing? I've read it through several times and still am not confident I have a good understanding.

Reply:
we're just getting a bag of key values out of some raw api call. the attempt at typing above doesn't really help us so just keep it as any. really this should be a real rpc so we get real types